net/freeradius-server/Makefile

```diff
@@ -0,0 +1,607 @@
+#
+# Copyright (C) 2008-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=freeradius-server
+PKG_VERSION:=3.0.11
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=\
+	ftp://ftp.freeradius.org/pub/freeradius/ \
+	ftp://ftp.freeradius.org/pub/freeradius/old/
+PKG_MD5SUM:=e8a748fe824223ed488cc74a1645023e
+PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
```

Comment:
hi @LucileQ, sorry for my brivety,
PKG_MAINTAINER is still set to @dangowrt, that's why i was asking the question.
We only merge package with a maintainer, and if nobody wants to maintain it I can't merge this PR
Just to be sure I'm not misreading, @LucileQ you don't want to maintain this package?